config/shapes/process_selection.py

```diff
@@ -346,16 +474,16 @@ def HWW_base_process_selection(channel, era):
 """
 
 
-# def DY_process_selection(channel, era):
-#     DY_process_weights = DY_base_process_selection(channel, era).weights
+# def DY_process_selection(channel, era, vs_jet_wp, vs_ele_wp):
+#     DY_process_weights = DY_base_process_selection(channel, era, vs_jet_wp, vs_ele_wp).weights
 #     DY_process_weights.append((
 #         "((genbosonmass >= 50.0)*6.2139e-05*((npartons == 0 || npartons >= 5)*1.0 + (npartons == 1)*0.1743 + (npartons == 2)*0.3556 + (npartons == 3)*0.2273 + (npartons == 4)*0.2104) + (genbosonmass < 50.0)*numberGeneratedEventsWeight*crossSectionPerEventWeight)","z_stitching_weight"))
 #     return Selection(name = "DY",
 #                      weights = DY_process_weights)
```

Comment:
why commented out?

Reply:
It is the leftover form `main` branch, there it's also commented, probably it's made for stitching but now we use inclusive samples only